rules/connect-client-im/rule.md

```diff
@@ -0,0 +1,68 @@
+---
+type: rule
+title: Do you know why it’s important to talk to clients on Teams? 
+uri: client-communication-im
+authors:
+  - title: Asher Paris
+  - url: https://ssw.com.au/people/
+related: rules-to-better-microsoft-teams
+created: 2023-10-31T16:50:21.000Z
+guid: 4332723e-c780-4d0c-a759-5067404ed8fc
+  
+---
+Effective communication with your clients on Microsoft Teams builds credibility, trust and ensures they receive the deserved attention.
+
+<!--endintro-->
+
+# Do you know why it’s important to talk with clients on Teams? 
+
+Microsoft Teams is an excellent channel for communicating with clients, especially at the early stage of client development, like after the initial meeting. It encourages the client to reach out and ask you anything about the business, an upcoming spec review and vice versa. This impresses the client and brings your business closer to a Specification Review.
+
+In other words, Microsoft Teams = buddies :-)
+
+## Microsoft Teams Context
+SSW has an open Microsoft 365 (tenant), which means any external email address can communicate with SSW via SSW Teams. However, some organisations have a closed tenant, meaning you can only communicate via Teams with their organisation’s consent.
+
+### Getting started
+Once you’ve arranged an Initial Meeting or before a Spec Review, follow these steps to begin communicating with your client on teams.
+
+1. Open Microsoft Teams
+2. Open a ‘New Client’ chat or press (Control + N)
+3. Enter the client’s email address
+4. Create the chat
+
+### If the client has an open tenant, you should send an open tenant request email.
+
+#### Open Tenant Email   
+Subject: Communication via Microsoft Teams
+
+Dear [client]
+
+I hope you’re well.
+
+As discussed earlier, we should use Microsoft Teams to chat because it ensures quick responses to your queries.
+
+No development question is too big or too small so don’t hesitate to reach out.
+
+Thanks
+
+[sign off]
```

Comment:
```suggestion
::: email-template  
|          |     |
| -------- | --- |
| To:      | {{ CLIENT }} |
| Subject: | Communication via Microsoft Teams |  
::: email-content  

### Dear {{ CLIENT }}

I hope you’re well.

As discussed earlier, we should use Microsoft Teams to chat because it ensures quick responses to your queries.

No development question is too big or too small so don’t hesitate to reach out.

Thanks

{{ SIGN OFF }}

{{ YOUR NAME }}
:::  
:::  
```